Build-OpenCV.sh

```diff
@@ -64,11 +64,10 @@ fi
 
 
 execute sudo apt-get update
-execute sudo apt-get install build-essential curl g++ cmake cmake-curses-gui git pkg-config checkinstall -y
-execute sudo apt-get install libopenblas-dev liblapacke-dev libatlas-base-dev gfortran -y
+execute sudo apt-get install build-essential curl g++ cmake cmake-curses-gui git pkg-config checkinstall gcc-8 g++-8 -y
```

Comment:
`gcc-8` on 16.04 requires `ubuntu-toolchain-r` PPA. I think it's okay if we don't add that since probably no one will be using this on a fresh 16.04 install now